app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -85,8 +85,8 @@ const RetweetWrapper = styled.div`
 `;
 
 export const TabBar = () => (
-  <svg width="960" height="4">
-    <rect width="960" height="1" fill={colors.white} opacity="0.2" />
+  <svg width="1020" height="4">
+    <rect width="1020" height="1" fill={colors.white} opacity="0.2" />
```

Comment:
I noticed on the projector that the twitter panel can go behind the tab bar. Is this causing that side effect? Ivan thght it would be best for the tweets to be constrained to the left of tabs.

Reply:
No, this is merely a divider between the previous and current tweets. This wouldn't cause that behavior. I can constrain it though